### PR TITLE
added more test cases for jQuery_event test cases.

### PR DESCRIPTION
**Pull request by taro1121**

### CODE REVIEW

File: jQuery_event/public/js/main.js [hunk 1]
```diff
@@ -1,38 +1,131 @@
 $(document).ready(function() {
-    $('.div').bind('click', function(event) {
-        alert('Hi there!');
 
-        //unbind so that the alert appear only once.
-        //$('.div #end').unbind('click');
+    //#1. Blur: msg. when unfocused from text box.
+    $('#blur').bind('blur', function(event) {
```
Comment (paragsm1) [inline jQuery_event/public/js/main.js]: Can you change the ID name? blur is not semantic

File: jQuery_event/public/js/main.js [hunk 2]
```diff
@@ -1,38 +1,131 @@
...
+    $('#dblclick').bind('dblclick', function(event) {
+        console.log('#4 DBLCLICK', $(this).text());
     });
 
-    $('#blur').bind('blur', function(event) {
-        console.log('BLUR', $(this).val());
+//#5(NOT-WORKING). error occurs when there is an error
+    // $(document).bind('error', function(event){
+    //     console.log('error happened in the document');
+    // });
```
Comment (paragsm1) [inline jQuery_event/public/js/main.js]: Instead og 'bind', can you try the patern like in this example - $( "img" )
  .error(function() {
    $( this ).attr( "src", "replacement.png" );
  })
  .attr( "src", "missing.png" );

File: jQuery_event/public/js/main.js [hunk 3]
```diff
@@ -1,38 +1,131 @@
...
+    // $(window).error(function(){
+    //     console.log('error happened in the document');
+    // });
+
```
Comment (paragsm1) [inline jQuery_event/public/js/main.js]: This is some more info about window.error that you may find useful for future - https://danlimerick.wordpress.com/2014/01/18/how-to-catch-javascript-errors-with-window-onerror-even-on-chrome-and-firefox/

File: jQuery_event/public/js/main.js [hunk 4]
```diff
@@ -1,38 +1,131 @@
...
+    // #9. Keyup occurs when key is released.
+    $('#blur').keyup(function(event) {
+        console.log('#9 You just keyup!');
+    });
+
+//10(NOT-WORKING). load occurs when document is laded.
+//     $(document).load(function(event) {
+//         console.log('You just loaded a document!');
+//     });
```
Comment (paragsm1) [inline jQuery_event/public/js/main.js]: only thing I can think of is - try running it outside document.ready block ... or better, use - $(window).load(function() { ... });

File: jQuery_event/public/js/main.js (hunk 1 above)
Comment (taro1121) [reply jQuery_event/public/js/main.js]: done in next checkin.

File: jQuery_event/public/js/main.js (hunk 2 above)
Comment (taro1121) [reply jQuery_event/public/js/main.js]: error() is depreciated.  I replace with {on("error", handler)} but still not working (spent 60+ min).

File: jQuery_event/public/js/main.js (hunk 3 above)
Comment (taro1121) [reply jQuery_event/public/js/main.js]: error() was depreciated i believe.

File: jQuery_event/public/js/main.js (hunk 4 above)
Comment (taro1121) [reply jQuery_event/public/js/main.js]: Fixed by $(window).one("load", function(){...});

**Pull request closed without merging**